### ISSUE TITLE
Accessibility issue when editing a collection type

### ISSUE THREAD

**Issue by brunoprietog** [strapi/strapi#8882]
### Describe the bug

The button to edit a collection type is not accessible.

### Steps to reproduce the behavior

1. Go to the admin panel
2. Plugins, Content-Types Builder
3. Create new collection type and/or enter one that has already been created.
4. If you try to access the button to edit with the keyboard you cannot, and it also does not have a label to be read by a screen reader.

### Expected behavior

The button should be accessible with the keyboard and read by a screen reader, such as NVDA.

### System

- Strapi version: Strapi v3.1.6 — Community Edition

Hopefully they can fix this, as I am blind and depend on strapi for 2 important projects.

Thank you very much.

**Comment by derrickmehaffy** [comment]
Closing in favor of: https://github.com/strapi/strapi/issues/8649

Thank you for the issue, we are currently evaluating our accessibility and don't currently have an ETA
